Getting Exception for Java-based (Scala) migration

I am using Play 2.3.8 and play-flyway 1.2.1. This is my database configuration in `application.conf`:

```
db.defaultdb.driver=org.h2.Driver
db.defaultdb.url="jdbc:h2:mem:play;DB_CLOSE_DELAY=-1"
db.defaultdb.user=sa
evolutionplugin=disabled
```

I have one single Java-based migration on `app/db/migration/defaultdb/V1__FirstMigration.scala` with these contents:

```
package db.migration.defaultdb

import org.flywaydb.core.api.migration.jdbc.JdbcMigration
import java.sql.Connection

class V1__FirstMigration extends JdbcMigration {
  override def migrate(conn: Connection) {
    conn.createStatement().executeUpdate(
      """create table language (
        |    id integer primary key,
        |    name varchar(100) not null
        |);""".stripMargin)
  }
}
```

Everything seems normal to me. However, when I start my application (upon the first request) I get this error message:

```
[info] play - database [defaultdb] connected at jdbc:h2:mem:play
[debug] application - Directory for migration files found. db/migration/defaultdb
[error] application - 

! @6ldg44m9c - Internal server error, for (GET) [/] ->

play.api.UnexpectedException: Unexpected exception[FileNotFoundException: Migration file not found. db.migration.defaultdb.V1__FirstMigration]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:166) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.Option.map(Option.scala:146) ~[scala-library-2.11.5.jar:na]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128) ~[play_2.11-2.3.8.jar:2.3.8]
Caused by: java.io.FileNotFoundException: Migration file not found. db.migration.defaultdb.V1__FirstMigration
	at com.github.tototoshi.play2.flyway.Plugin$$anonfun$com$github$tototoshi$play2$flyway$Plugin$$migrationDescriptionToShow$3.apply(Plugin.scala:117) ~[play-flyway_2.11-1.2.1.jar:1.2.1]
	at com.github.tototoshi.play2.flyway.Plugin$$anonfun$com$github$tototoshi$play2$flyway$Plugin$$migrationDescriptionToShow$3.apply(Plugin.scala:117) ~[play-flyway_2.11-1.2.1.jar:1.2.1]
	at scala.Option.getOrElse(Option.scala:121) ~[scala-library-2.11.5.jar:na]
	at com.github.tototoshi.play2.flyway.Plugin.com$github$tototoshi$play2$flyway$Plugin$$migrationDescriptionToShow(Plugin.scala:117) ~[play-flyway_2.11-1.2.1.jar:1.2.1]
	at com.github.tototoshi.play2.flyway.Plugin$$anonfun$com$github$tototoshi$play2$flyway$Plugin$$checkState$1$$anonfun$apply$8.apply(Plugin.scala:126) ~[play-flyway_2.11-1.2.1.jar:1.2.1]
```

Is this a bug? Or did I miss some step?

--- a/plugin/src/main/scala/com/github/tototoshi/play2/flyway/Plugin.scala
+++ b/plugin/src/main/scala/com/github/tototoshi/play2/flyway/Plugin.scala
@@ -110,7 +110,7 @@ class Plugin(implicit app: Application) extends play.api.Plugin
           |${readInputStreamToString(in)}""".stripMargin
     }.orElse {
       import scala.util.control.Exception._
-      allCatch opt { Class.forName(migration.getScript) } map { cls =>
+      allCatch opt { app.classloader.loadClass(migration.getScript) } map { cls =>
         s"""|--- ${migration.getScript} ---
             | (Java-based migration)""".stripMargin
       }

